sdk/resourcemanager/Azure.ResourceManager.Core/src/Custom/Resources/PredefinedTagDeleteOperation.cs

```diff
@@ -11,7 +11,7 @@
 using Azure.Core;
 using Azure.Core.Pipeline;
 
-namespace Azure.ResourceManager.Core
+namespace Azure.ResourceManager.Resources.Models
```

Comment:
Shouldn't this be Azure.ResourceManager.Resources? this is not inside Models.

Reply:
[LRO objects are inside models in data plane as well](https://apiview.dev/Assemblies/Review/06c0eac8cadd4fb8881ada2cb925fedb#Azure.Storage.Blobs.Models.CopyFromUriOperation) since they are return objects  only and we want the main namespace to highlight the main classes you will interact with

Reply:
Ahh, if that's the case then should we move these to `/sdk/resourcemanager/Azure.ResourceManager.Core/src/Custom/Resources/Models` folder?